sdks/python/src/opik/evaluation/scores_logger.py

```diff
@@ -1,12 +1,14 @@
-from typing import List
+from typing import List, Optional
 from opik.types import FeedbackScoreDict
 
 from opik.api_objects import opik_client
 from . import test_result
 
 
 def log_scores(
-    client: opik_client.Opik, test_results: List[test_result.TestResult]
+    client: opik_client.Opik,
+    test_results: List[test_result.TestResult],
+    project_name: Optional[str] = None,
```

Comment:
It should be just  `project_name: Optional[str]` without the default value.
There is no real need for the default value because we call this function in one place only and pass `project_name` explicitly.
Such arguments with the default value in the middle of the processing pipeline often lead to unnoticed bugs.